Gemfile.lock

```diff
@@ -231,4 +231,4 @@ DEPENDENCIES
   fastlane (>= 2.98.0)
 
 BUNDLED WITH
-   1.16.6
+   2.0.1
```

Comment:
(happy to revert this if there are any objections — not sure what sentry's stances are on pinning dependencies)

Reply:
This is fine 👍